homeassistant/components/formula_one/const.py

```diff
@@ -0,0 +1,42 @@
+"""Constants for the Formula 1 integration."""
+
+from datetime import timedelta
+
+DOMAIN = "formula_one"
+
+DEFAULT_UPDATE_INTERVAL = timedelta(hours=1)
+
+F1_STATE_UNAVAILABLE = "(unavailable)"
```

Comment:
Home Assistant has an unavailable state (when the state returns `None`), please don't invent/add new ones.